### PR TITLE
Add potal UI helm chart

### PR DESCRIPTION
**Pull request by mnaghavi**

### CODE REVIEW

File: portal/values.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  enabled: false
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+appConfig:
+  port: 80
```
Comment (andricDu) [inline portal/values.yaml:52]: this value does not appear to be used in your deployment.

File: portal/values.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Default values for portal.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: nginx
```
Comment (andricDu) [inline portal/values.yaml:8]: even though we are currently using the private registry, we can just use the name of the image we create for now. I'm assuming it's something like `portal`.

**Review by andricDu — CHANGES_REQUESTED** [review]

File: portal/values.yaml (hunk 1 above)
Comment (andricDu) [reply portal/values.yaml:52]: Looks like this is not even configurable for our portal image if you inspect the docker file.

Comment (mnaghavi) [reply portal/values.yaml:52]: Yes, you're right. removed

File: portal/values.yaml (hunk 2 above)
Comment (mnaghavi) [reply portal/values.yaml]: That makes sence. changed.

**Review by andricDu — APPROVED** [review]

**Pull request merged**